source/_includes/puppet-collections/_pc_versions.html

```diff
@@ -0,0 +1,12 @@
+<table>
+  <thead>
+    <tr>
+      <th>Puppet Collection</th> <th>puppet-agent</th> <th>puppetserver</th> <th>puppetdb</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr><td><a href="/puppet/4.3/reference/puppet_collections.html#PC-1">PC 1</a></td> <td><a href="/puppet/4.3/reference/about_agent.html">1.3.2</a></td> <td><a href="/puppetserver/2.2/release_notes.html">2.2.0</a></td> <td><a href="/puppetdb/3.2/">3.2</a></td></tr>
+  </tbody>
+</table>
+
```

Comment:
This is a skeleton of a potential HTML table for the references generator, @nfagerlund, as prep for being able to autogenerate information about Puppet Collection components as we do for things like the `puppet-agent` package.